.travis.yml

```diff
@@ -30,7 +30,7 @@ env:
     - "TESTLANG=D"
     - "TESTLANG=Dart"
     - "TESTLANG=Dylan"
-    - "TESTLANG=Elixir"
+    - "TESTDIR=Elixir/plug"
```

Comment:
Can you undo this change? Phoenix still lives in this folder and needs to be tested. The removed line will test all frameworks inside the Elixir folder so no need to update.

Reply:
The issue is the Phoenix tests are failing at least on my local they are.

Reply:
I get that. Phoenix shouldn't be tested in this PR. I've written a script that only tests the frameworks of the files being touched so your PR will still pass. We need to make sure the travis.yml file covers all possible tests though as it's a master list.

Reply:
I was about to open another PR to fix those. Would you rather it just be all one PR?

Reply:
That works fine for me

Reply:
Ok will switch it to the folder only.